aws-ec2-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -123,36 +123,33 @@ management_worker_rpm_source_url: 'https://gigaspaces-repository-eu.s3.amazonaws
 #############################
 #influxdb_endpoint_ip: ''
 
-
-###########################
+#############################
 # Offline Resources Upload
-###########################
+#############################
 # You can configure a set of resources to upload at bootstrap. These resources
-# will reside on the manager and enable offline deployment. dsl_resources
-# should contain any resource needed in the parsing process (i.e. plugin.yaml files),
-# and any plugin archive should be compiled through the designated wagon tool,
-# and the path should be passed to plugin_resources. Any resource your
-# blueprint might need, could be upload trough this mechanism.
+# will reside on the manager and enable offline deployment. `dsl_resources`
+# should contain any resource needed in the parsing process (i.e. plugin.yaml files)
+# and any plugin archive should be compiled using the designated wagon tool
+# which can be found at: http://github.com/cloudify-cosmo/wagon.
+# The path should be passed to plugin_resources. Any resource your
+# blueprint might need, could be uploaded using this mechanism.
+
 #dsl_resources:
 #  - {'source_path': '/cfy/cloudify/plugins/fabric-plugin/plugin.yaml', 'destination_path': '/fabric-plugin/1.3rc1/plugin.yaml'}
```

Comment:
source links should point to S3, not file path. maxim changed these in the manager blueprints themselves (see here https://github.com/cloudify-cosmo/cloudify-manager-blueprints/blob/master/aws-ec2-manager-blueprint.yaml#L382 ) but apparently forgot in the inputs files.